tests/integration_tests/key_value/api_tests.py

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# isort:skip_file
+import json
+
+from superset import db
+from superset.models.key_value import KeyValue
+from tests.integration_tests.base_tests import SupersetTestCase
+
+duration_ms = 10000
+reset_duration_on_retrieval = True
+value = "test"
+
+
+class KeyValueTests(SupersetTestCase):
+    def post(self):
+        payload = {
+            "duration_ms": duration_ms,
+            "reset_duration_on_retrieval": reset_duration_on_retrieval,
+            "value": value,
+        }
+        resp = self.client.post("api/v1/key_value_store/", json=payload)
+        data = json.loads(resp.data.decode("utf-8"))
+        return data.get("key")
+
+    def setUp(self):
+        self.login(username="admin")
+        rows = db.session.query(KeyValue).all()
+        for row in rows:
+            db.session.delete(row)
+        db.session.commit()
+
+    def test_post(self):
+        key = self.post()
+        result = db.session.query(KeyValue).first()
+        self.assertEqual(key, str(result.key))
```

Comment:
Nit: `pytest` works better with naked asserts, eg:

```suggestion
        assert key == str(result.key)
```

https://docs.pytest.org/en/latest/how-to/assert.html#assert